consensus/src/block_storage/sync_manager.rs

```diff
@@ -443,6 +449,28 @@ impl BlockRetriever {
         monitor!("retrieve_block_for_id_chunk", {
             let mut interval = time::interval(retry_interval);
             let mut futures = FuturesUnordered::new();
+            if retrieve_batch_size == 1 {
```

Comment:
Why do this only for 1 batch size?

Reply:
we only do multi block fetch during the fast forward sync process, which seems unlikely to hit the local buffer